examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<p>A few comments:</p><p> </p><ul><li>Possibly replace the hand-drawn wave and seabed lines with some kind of smooth curve? I think Bezier curves or similar would look a lot neater and cleaner. Really good figure otherwise.</li><li>Do either of the links you refer to have a DOI?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thx for the review, first of all! I agree, I will make a better drawing on inkscape and update the notebook. About the DOI, not that I'm aware of.</p>

Reply:
Is there any paper which outlines them or uses a comparable implementation (even if it's more complex than the examples shown here)? Not sure how much anyone else is bothered, but I think it would be good to be able to link to something with a DOI (so we don't get any broken links down the line)

Reply:
<p>Well, I could use an internet archive link, what do u think?</p>

Reply:
Tbh, if there isn't an easy replacement than I'd just let it slide and change it if someone complains that the link is dead later., assuming no-one else is bothered by that